cubids/utils.py

```diff
@@ -122,10 +129,21 @@ def _file_to_entity_set(filename):
     str
         A set of entities extracted from the filename.
 
+    Notes
+    -----
+    This function relies on the variable NON_KEY_ENTITIES from cubids.constants.
+    This is a set of entities to ignore in the entity set.
+    The constant may be modified by the CuBIDS class, which will remove "session"
+    if is_longitudinal is True and acq_group_level is "session".
+
     Examples
     --------
     >>> _file_to_entity_set("sub-01_ses-01_task-rest_bold.nii.gz")
     'session-01_suffix-bold_task-rest'
+
+    Field maps will have an extraneous "fmap" entity.
```

Comment:
Do we know what this is for? Should we warn users not to mess with it?

Reply:
I don't know why they have it, but I'm guessing it makes it easier to query field maps for a downstream tool, like fMRIPrep. I'm happy to add something to the documentation about it, but I'm not sure where to do that.